includes/widgets/accordion.php

```diff
@@ -299,6 +299,7 @@ protected function register_controls() {
 				'type' => Controls_Manager::COLOR,
 				'selectors' => [
 					'{{WRAPPER}} .elementor-accordion-icon, {{WRAPPER}} .elementor-accordion-title' => 'color: {{VALUE}};',
+					'{{WRAPPER}} .elementor-accordion-icon svg' => 'fill: {{VALUE}};',
```

Comment:
Why don't you use a CSS variable instead?

Reply:
As agreed, keeping it as the rest of the file.